cli/cli/commands/import/import.go

```diff
@@ -241,6 +251,36 @@ func convertComposeProjectToStarlark(compose *types.Project) (string, map[string
 	return script, requiredFileUploads, nil
 }
 
+func getMemoryMegabytesReservation(deployConfig *types.DeployConfig) int {
+	if deployConfig == nil {
+		return 0
+	}
+	reservation := 0
+	if deployConfig.Resources.Reservations != nil {
```

Comment:
is `Resources` bound to be set or can `Resources` be nil?

Reply:
lets add a test case for this?

Reply:
It is a value not a pointer, so per type system it must be set!